src/config.rs

```diff
@@ -23,6 +23,6 @@ mod test {
 
     #[test]
     fn test_reading_config() {
-        let cfg = dbg!(Config::load());
+        let _cfg = dbg!(Config::load());
```

Comment:
ale wiesz że testowanie polega na czymś więcej niż tylko wypisywanie wartości w terminalu? 😂

Reply:
Nieprawda, a jak testy fejlują to co robią? Wypisują szajs, który później jest parsowany przez inne readery ;)

Ten szajs jest parsowany przeze mnie - mój projekt, moje reguły QA :D